src/android/Peripheral.java

```diff
@@ -527,8 +565,22 @@ public void onCharacteristicWrite(BluetoothGatt gatt, BluetoothGattCharacteristi
             if (writeCallback != null) {
                 if (status == BluetoothGatt.GATT_SUCCESS) {
                     writeCallback.success();
+                    Bundle bundle = new Bundle();
```

Comment:
This function will get called whenever we are updating/writing any characteristic value. Tracking success here will flood our analytics with lots of data that won't be useful for any analytics. We might be benefited with BT_WRITE_FAILURE though but only if we include information about characteristic getting updated it will not allow us any diagnostics.